deploy/roles/dockerized_service/tasks/main.yml

```diff
@@ -0,0 +1,101 @@
+- name: volumes exist
+  docker_volume:
+    name: "{{ item }}"
+  with_items: "{{ docker_volumes }}"
+
+
+- name: configs updated
+  block:
+    - name: config temporary dir
+      tempfile:
+        state: directory
+      register: config_volume_mount
+      changed_when: no
+      check_mode: no
+
+    - name: config volume proxy
+      docker_container:
+        name: "{{ config_volume_proxy_container_name }}"
+        image: busybox
+        detach: yes
+        tty: yes
+        auto_remove: yes
+        volumes:
+          - "{{ config_volume }}:/v"
+      changed_when: no
+      check_mode: no
+
+    - name: extract config from volume
+      command:
+        cmd: "docker cp {{ config_volume_proxy_container_name }}:/v {{ config_volume_mount.path }}"
+      changed_when: no
+      check_mode: no
```

Comment:
why do we need this? is this just so we can create volume and work with it?

Reply:
Se can create volume, but it will be empty. This (dirty) approach solves the following problems:
- use named volumes without trashing random directories on host
- see diffs of what's put into a config_volume